cmd/up/up.go

```diff
@@ -379,12 +380,53 @@ func loadManifestOverrides(dev *model.Dev, upOptions *UpOptions) error {
 		dev.LoadForcePull()
 	}
 
+	if len(upOptions.Envs) > 0 {
+		if err := setEnvVarsFromCmd(dev, upOptions); err != nil {
+			return err
+		}
+	}
+
 	dev.Username = okteto.Context().Username
 	dev.RegistryURL = okteto.Context().Registry
 
 	return nil
 }
 
+func setEnvVarsFromCmd(dev *model.Dev, upOptions *UpOptions) error {
```

Comment:
We could narrow down a bit the parameters, and (this is more of a personal preference) it could be more functional. The signature would look something like `func getEnvVarsFromCmd(devEnv model.Environment, commandEnvVariables []string) (model.Environment, error)`

Reply:
after discussing this approach we have decided to adopt the more functional approach proposed